rancher/distros-test-framework
# Version bump updates

#### Proposed Changes ####





#### Types of Changes ####
 
- debug mode added on template
- fixed 2 issues related to concurrency  
- Here we was unnecessaryly spinning up routines based on CMDS , which with a lot of commands and more than 1 ip, started to flood ssh and returning error for handshake 
* src/github.com/rancher/distros-test-framework/shared/aux.go:57 ERRO[2024-02-21T15:03:58Z] ssh: handshake failed: EOF Last call: github.com/rancher/distros-test-framework/shared.configureSSH in file:/go/src/github.com/rancher/distros-test-framework/shared/aux.go:182 ERRO[2024-02-21T15:03:58Z] ssh: handshake failed: EOF

https://github.com/rancher/distros-test-framework/blob/3311bc26a363a4418f3a05eeafebc032d2fb49e9/pkg/template/processor.go#L36 


 - fixed an issue here https://github.com/rancher/distros-test-framework/blob/3311bc26a363a4418f3a05eeafebc032d2fb49e9/pkg/template/processor.go#L67
which was sending a cmd that returns  the node info not parsed correctly , causing  to pass by this info to next function returning error related to not finding the host 

- fixed general error formatting wrapper  on the PKG 
- added a job that runs most of all general bumps tested
 
- fixed expectedValue result formatting 




#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
yes - 

2. Verify code lint; we should not have errors.
yes - 

3. Update the documentation if needed.
yes- 

4. Update makefile and docker run if adding new tests.
yes 

5. Run your tests at least 4 times with all configurations needed and possible.
yes 

6. If needed test with different os types.

OS TESTED

Red Hat Enterprise Linux 8.9 (Ootpa)

Red Hat Enterprise Linux  9.2 

 Rocky Linux 8.9 (Green Obsidian)

 Oracle Linux Server 9.3 
ORACLE 8.9


SUSE Linux Enterprise Server 15 SP4


ubuntu

#### Linked Issues ####



#### Further Comments ####
